src/fileio.cpp

```diff
@@ -289,14 +289,63 @@ void MakeDirectory(const char *path)
 	NOT_REACHED();
 }
 
+/**
+ * Copy a file.
+ * @param src Source file.
+ * @param dest Destination file.
+ */
+void CopyBinaryFile(const char *src, const char *dest)
+{
+	FILE *in_file = nullptr;
+	in_file = fopen(src, "rb");
+	if (in_file == nullptr) {
+		fprintf(stderr, "Could not open file for reading: %s\n", src);
+		NOT_REACHED();
+	}
+
+	FILE *out_file = nullptr;
+	out_file = fopen(dest, "wb");
+	if (out_file == nullptr) {
+		fprintf(stderr, "Could not open file for writing: %s\n", dest);
+		NOT_REACHED();
+	}
+
+	int byte;
+	while ((byte = getc(in_file)) != EOF) putc(byte, out_file);
```

Comment:
You sneaked an assignment into the condition.
There is another one as well.

Code is probably slow for copying files (faster would be to read/write blocks), but since the data isn't big, I don't see that as a problem.